packages/web/src/lib/myalgowallet-mode.ts

```diff
@@ -57,8 +57,8 @@ export class MyAlgoWalletSession {
 	accounts: Accounts[] = [];
 	addresses: Address[] = [];
 
-	constructor(chain: string, connector?: MyAlgoConnect) {
-		this.algodClient = algoexplorerAlgod(chain);
+	constructor(walletURL: any, connector?: MyAlgoConnect) {
```

Comment:
Should we create a type for `walletURL`?

Reply:
Sure let me add type for walletURL.